tools/speech_data_explorer/README.md

```diff
@@ -7,6 +7,7 @@ Features:
 - dataset's statistics (alphabet, vocabulary, duration-based histograms)
 - navigation across dataset (sorting, filtering)
 - inspection of individual utterances (waveform, spectrogram, audio player)
+- errors' analysis if a manifest contains 'pred_text' field (WER, CER, Word Match Rate, Mean Word Accuracy, diff)
```

Comment:
Is there any place in the docs we describe our manifest format and the expected columns? It would be nice to have a link here or maybe explicitly state that ‘text’, ‘duration’, ‘audio_filepath’ columns are required for the explorer to work?